proposals/xxxx-package-manager-branch-support.md

```diff
@@ -0,0 +1,48 @@
+# Package Manager Support for branches in the package manifest
```

Comment:
I would leave off "in the package manifest", as that's a detail of our proposed solution, which is up for discussion. Maybe "Package Manager Support for Branch-Based Development"